package.json

```diff
@@ -1,28 +1,37 @@
 {
   "name": "storeon-observable",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "description": "RxJS based module for Storeon which allows to create async actions",
-  "main": "dist/index.umd.js",
-  "module": "dist/index.es5.js",
-  "typings": "dist/types/index.d.ts",
+  "main": "index.js",
+  "module": "module/index.js",
+  "typings": "index.d.ts",
   "files": [
     "dist"
   ],
   "repository": "git@github.com:distolma/storeon-observable.git",
   "author": "Dmytro Mostovyi <djmakaron2009@gmail.com>",
+  "contributors": [
+    "Dmytro Mostovyi <djmakaron2009@gmail.com>",
+    "Paweł Majewski <majo44@gmail.com>"
+  ],
   "license": "MIT",
   "scripts": {
     "start": "rollup -c rollup.config.ts -w",
-    "build": "rollup -c rollup.config.ts",
-    "lint": "eslint *.js",
-    "test": "jest --coverage && yarn lint && size-limit"
+    "build": "tsc && tsc -p tsconfig.module.json && size-limit",
+    "lint": "eslint index.ts",
+    "test": "jest --coverage && yarn lint"
```

Comment:
Better run build before size-limit, imho

```sh
jest --coverage && yarn lint && npm build && size-limit
```

Reply:
I changed the scripts, so now build is main script which orchestrate all other